python/python-playground-questions/adding-and-sorting-lists/adding-and-sorting-lists.md

```diff
@@ -26,9 +26,13 @@ setupCode:
 
 ## Content
 
-Can you make a program that will take two numbered lists, join them together into one list and then sort them by size?
+> 👩‍💻 Your task is to make a program that will:
+> - **take two numbered lists**
+> - **join them together into one list**
+> - **sort them by size**
```

Comment:
Since it was mentioned to sort after joining it is referring to the new list. Regardless here is the new copy:
> 👩‍💻 Your task is to: **create a function that joins two lists and sorts the resulting list in ascending order.**

Reply:
> Since it was mentioned to sort after joining it is referring to the new list.

mb, my original review wasn't clear enough. my confusion came from the fact that you said "sort **them** by size", so that makes me think we should sort the two arrays